test/unit_tests/braket/circuits/test_ascii_circuit_diagram.py

```diff
@@ -29,6 +29,10 @@ def test_empty_circuit():
     assert AsciiCircuitDiagram.build_diagram(Circuit()) == ""
 
 
+def test_only_gphase_circuit():
+    assert AsciiCircuitDiagram.build_diagram(Circuit().gphase(0.1)) == ""
```

Comment:
Hm are we sure about this? I think this could make sense, but I'd appreciate hearing a bit of context of why we're convinced